pkg/api/v1/types.go

```diff
@@ -363,6 +363,17 @@ type VirtualMachineInstanceMigrationState struct {
 	Failed bool `json:"failed,omitempty"`
 	// The VirtualMachineInstanceMigration object associated with this migration
 	MigrationUID types.UID `json:"migrationUid,omitempty"`
+	// The time for GiB of data to wait for the migration to be completed before aborting it
+	CompletionTimeoutPerGiB int64 `json:"completionTimeoutPerGiB,omitempty"`
```

Comment:
we could source the MigrationConfig here as a nested object in the VirtualMachineInstanceMigrationState struct. Then we could deepcopy the migration.Spec.Config to the vmi.Status.MigrationState.Config without having to define the values in both the migration config and VMI migration state.

```
type VirtualMachineInstanceMigrationState struct {
.
.
.
        Config v1.MigrationConfig `json:"migrationConfig,omitempty"`
}
```

would there be any drawbacks from doing it that way?

Reply:
:) thanks